### PR TITLE
feat: add buttons controls

### LINKED ISSUE #30: Builtin control components
- [x] Text input
- [x] Textarea
- [x] Number input
- [x] Checkbox
- [x] Select
- [x] Button group
- [x] Radio list
- [x] Checkbox list
- [x] Slider
- [ ] Date/hour
- [ ] Color picker
- [x] JSON code editor

### PR DESCRIPTION
**Pull request by hugoattal**
### Description

Add Button and ButtonGroup control components

image

image

### What is the purpose of this pull request? 

- [ ] Bug fix
- [x] New Feature
- [ ] Documentation update
- [ ] Other

### CODE REVIEW

File: packages/histoire-app/src/app/style/main.pcss [hunk 1]
```diff
@@ -53,6 +53,11 @@ html {
   font-size: 1rem;
 }
 
+input, button, textarea {
```
Comment (Akryum) [inline packages/histoire-app/src/app/style/main.pcss:56]: This can leak into userspace

File: packages/histoire-app/src/app/style/sandbox.css [hunk 2]
```diff
@@ -14,6 +14,11 @@ html {
   font-family: 'Lato', sans-serif;
 }
 
+input, button, textarea {
```
Comment (Akryum) [inline packages/histoire-app/src/app/style/sandbox.css:17]: See above

**Review by Akryum — CHANGES_REQUESTED** [review]

File: packages/histoire-controls/src/components/HstWrapper.vue [hunk 3]
```diff
@@ -5,15 +5,24 @@ export default {
 </script>
 
 <script lang="ts" setup>
+import { withDefaults, computed } from 'vue'
 import { VTooltip as vTooltip } from 'floating-vue'
 
-defineProps<{
+const props = withDefaults(defineProps<{
   title?: string
-}>()
```
[the remark this answers is not in the document]
Comment (hugoattal) [reply packages/histoire-controls/src/components/HstWrapper.vue:12]: Right, I should have gone for "isLabel" maybe 🤔... Anyway, here's "tag" : https://github.com/histoire-dev/histoire/pull/312/commits/f815b11302cd82112ad269172efc2c57a7880a5c

**Comment by Akryum** [comment]
I updated the design to be more like the mockup in #30 and match the other controls look more.
image

**Pull request merged**